src/applications/check-in/travel-claim/pages/error/index.jsx

```diff
@@ -0,0 +1,186 @@
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+
+import { makeSelectError, makeSelectTravelClaimData } from '../../../selectors';
+import Wrapper from '../../../components/layout/Wrapper';
+import ExternalLink from '../../../components/ExternalLink';
+
+const Error = () => {
+  const { t } = useTranslation();
+  const selectError = useMemo(makeSelectError, []);
+  const { error } = useSelector(selectError);
+
+  // @TODO refactor using something else from redux
+  const selectTravelClaimData = useMemo(makeSelectTravelClaimData, []);
+  const appointments = useSelector(selectTravelClaimData);
+  const appointmentDateTime =
+    appointments.length > 0 ? new Date(appointments[0].startTime) : null;
+
+  let alerts = [];
+  let header = '';
+
+  const FindOutLink = () => (
+    <ExternalLink
+      href="/health-care/how-to-file-a-va-travel-reimbursement-claim-online/"
+      hrefLang="en"
+      eventId="clicked-how-to-file-link-from-error"
```

Comment:
small nitpick and the figma doesn't specify, but it seems like other link events have "clicked" at the end of the string rather than the front. https://docs.google.com/spreadsheets/d/1BPOlI6vzH-tMTswvUrci3Q0NkBmfIIfo8Cgj_nM10gk/edit#gid=2075861504